config/routes.js

```diff
@@ -1,7 +1,9 @@
 // eslint-disable-next-line new-cap
 const router = require('express').Router();
 const indexCtrl = require('../app/controllers/index');
```

Comment:
Yeah actually using longhand as below is more legible for people? So make this `indexController` perhaps?